docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -1,31 +1,93 @@
-Teleport's plugins use the `access-plugin` user to manage access requests. In order to act as
-the user, the plugin must present the Teleport Auth Service with valid identity files.
-We export the identity files for the user using [`tctl auth sign`](../../setup/reference/cli.mdx#tctl-auth-sign).
+Like all Teleport users, `access-plugin` needs signed credentials in
+order to connect to your Teleport cluster. You will use the `tctl auth sign`
+command to request these credentials for your plugin. 
+
+<ScopedBlock scope={["oss", "enterprise"]}>
```

Comment:
I'm not sure, do we really support access request plugins in OSS? Only local `tctl` can actually approve the requests.

Reply:
We don't support access request plugins in OSS. I just didn't want users with the default scope (which is `oss` now) to see big chunks of missing content. Once we fix [this issue](https://github.com/gravitational/docs/issues/118), we can  remove `oss` here, I think.